content/react.md

```diff
@@ -238,6 +238,73 @@ Although this `ListsShowPage` container is intended to be instantiated by the Re
 <ListsShowPage params={{id: '7'}}/>
 ```
 
+<h3 id="using-tracker-component">Using `Tracker.Component`</h3>
+
+Once you've run `npm i --save tracker-component`, you'll be able to create components with the `Tracker.Component` class, it allows you to create a [container component](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.by86emv9b) which provides data to your presentational components.
+
+> Note that "container components" are analogous to the "smart components" and "presentational components" to the "reusable components" in the pattern we document in the [UI/UX article](http://guide.meteor.com/ui-ux.html#components), if you'd like to read more about how this philosophy relates to Meteor.
+
+For example, in the Todos example app, we have a `ListsShow` component, which renders list metadata and the tasks in the list. In order to do so, it needs to [subscribe](data-loading.html#subscriptions) to the `todos.inList` publication, check that subscription's readiness, then fetch the list of todos from the `Todos` collection.
+
+It also needs to be responsive to reactive changes in the state of those actions (for instance if a todo changes due to the action of another user). All this data loading complexity is a typical use-case for a container-presentational component split, and `Tracker.Component` class makes it simple to do this.
+
+We simply define the `ListsShow` component as a presentational component that expects its data to be passed in using React `props`:
+
+```js
+import React from 'react';
+
+export default class ListsShow extends React.Component {
+  ...
+}
+
+ListsShow.propTypes = {
+  list: React.PropTypes.object,
+  todos: React.PropTypes.array,
+  loading: React.PropTypes.bool,
+  listExists: React.PropTypes.bool,
+};
+```
+
+Then we create a `ListsShowPage` container component which wraps it and provides a data source:
+
+```js
+import { Meteor } from 'meteor/meteor';
+import { Lists } from '../../api/lists/lists.js';
+import Tracker from 'tracker-component';
```

Comment:
Is it weird that this overrides `Tracker` from `meteor/tracker`? If you have code like `Tracker.autorun` etc. seems like it wouldn't work anymore?

In general, I feel like making the package attach itself to the `Tracker` namespace doesn't make sense. Why not just `import TrackerComponent from 'tracker-component'` instead?